### DISCUSSION TITLE
❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by PhyllisJi** [snakers4/silero-models#80] [General]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

Is there a specific network structure used by English models?
Thx！

**Comment by snakers4** [comment]
https://github.com/snakers4/silero-models/discussions/36

https://github.com/snakers4/silero-models/discussions/12

**Reply by snakers4** [reply]
The models themselves are public
